### PR TITLE
New package: K8stooling.Kubazulo version 0.0.8-beta

### PR DESCRIPTION
**Pull request by maikbrauer**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/135352)

### CODE REVIEW

**Comment by maikbrauer** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 2:11:39 PM: 2024-01-22 14:11:38.145 [FAIL] Installer failed security check. Url: https://github.com/k8stooling/kubazulo/releases/download/v0.0.8-beta/kubazulo.exe Result: 0x80004005

(Automated response - build 623.)

**Comment by stephengillie** [comment]
Hi @maikbrauer,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 623.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 8:57:45 AM: 2024-01-23 08:57:45.025 [CLI ] Package hash verification failed. SHA256 in manifest [6f771ddd2ec8db37a91fa0af205948ff07ac1ed0af7d8c7ef5015cf1c0d6c01a] does not match download [3399f1dbd03105042394a7723c97da2f35855c50026e449185e4e0bca333ae08]

(Automated response - build 629.)

File: manifests/k/K8stooling/Kubazulo/0.0.8-beta/K8stooling.Kubazulo.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: K8stooling.Kubazulo
+PackageVersion: 0.0.8-beta
+InstallerType: portable
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/k8stooling/kubazulo/releases/download/v0.0.8-beta/kubazulo.exe
+  InstallerSha256: 6F771DDD2EC8DB37A91FA0AF205948FF07AC1ED0AF7D8C7EF5015CF1C0D6C01A
```
Comment (stephengillie) [inline manifests/k/K8stooling/Kubazulo/0.0.8-beta/K8stooling.Kubazulo.installer.yaml:10]:
```suggestion
  InstallerSha256: 3399F1DBD03105042394A7723C97DA2F35855C50026E449185E4E0BCA333AE08
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**